docs/localdatetime.md

```diff
@@ -0,0 +1,1242 @@
+ldt = Temporal.LocalDateTime.from(props)
+  // => 2020-11-01T01:30-07:00[America/Los_Angeles] ('compatible' is the default)
+earlier = Temporal.LocalDateTime.from(props, { disambiguation: 'earlier' })
+  // => 2020-11-01T01:30-07:00[America/Los_Angeles] ('earlier' is same as 'compatible' for backwards transitions)
+later = Temporal.LocalDateTime.from(props, { disambiguation: 'later' })
+  // => 2020-11-01T01:30-08:00[America/Los_Angeles] (same clock time, but one hour later -08:00 offset indicating Standard Time)
+later.toDateTime().difference(earlier.toDateTime())
+  // => PT0S  (same clock time...
+later.difference(earlier)
+  // => PT1H   ... but 1 hour later in real-world time)
+
+// Different disambiguation modes for times in the skipped clock hour after DST starts in the Spring
+// Offset of -07:00 is Daylight Saving Time, while offset of -08:00 indicates Standard Time.
+props = { timeZone: 'America/Los_Angeles', year: 2020, month: 3, day: 8, hour: 2, minute: 30 }
+ldt = Temporal.LocalDateTime.from(props, { disambiguation: 'compatible' })
+  // => 2020-03-08T03:30-07:00[America/Los_Angeles]
+ldt = Temporal.LocalDateTime.from(props)
+  // => 2020-03-08T03:30-07:00[America/Los_Angeles] ('compatible' is the default)
+earlier = Temporal.LocalDateTime.from(props, { disambiguation: 'earlier' })
+  // => 2020-03-08T01:30-08:00[America/Los_Angeles] (1:30 clock time; still in Standard Time)
+later = Temporal.LocalDateTime.from(props, { disambiguation: 'later' })
+  // => 2020-03-08T03:30-07:00[America/Los_Angeles] ('later' is same as 'compatible' for backwards transitions)
+later.toDateTime().difference(earlier.toDateTime())
+  // => PT2H  (2 hour difference in clock time...
+later.difference(earlier)
+  // => PT1H   ... but 1 hour later in real-world time)
+```
+
+### Temporal.LocalDateTime.**compare**(_one_: Temporal.LocalDateTime, _two_: Temporal.LocalDateTime) : number
+
+**Parameters:**
+
+- `one` (`Temporal.LocalDateTime`): First value to compare.
+- `two` (`Temporal.LocalDateTime`): Second value to compare.
+
+**Returns:** an integer indicating whether `one` comes before or after or is equal to `two`.
+
+- Zero if all fields are equivalent, including the calendar ID and the time zone name.
+- &minus;1 if `one` is less than `two`
+- 1 if `one` is greater than `two`.
+
+Comparison will use absolute time, not clock time, because sorting is almost always based on when events happened in the real world, but during the hour before and after DST ends in the fall, sorting of clock time will not match the real-world time sort order.
+
+If absolute timestamps are equal, then `.calendar.id` will be compared alphabetically.
+If those are equal too, then `.timeZone.name` will be compared alphabetically.
+Even though alphabetic sort carries no meaning, it's used to ensure that unequal instances have a deterministic sort order.
+
+This function can be used to sort arrays of `Temporal.LocalDateTime` objects.
```

Comment:
Move this line up to be the first line after the "Returns" paragraph / list.

Reply:
good idea. moved.